src/Json.Schema.ToDotNet.UnitTests/DataModelGeneratorTests.cs

```diff
@@ -1070,6 +1071,7 @@ public SNodeKind SNodeKind
         /// </summary>
         [DataMember(Name = ""intPropWithDefault"", IsRequired = false, EmitDefaultValue = false)]
         [DefaultValue(42)]
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
```

Comment:
>DefaultValueHandling [](start = 22, length = 20)

What's your process for maintaining these tests? Is it only manually? Do you use debugger values for comparison? Do you ever write them to disk and diff? Etc.

Signed, 'Harps on a Consistent Set of Strings'

Reply:
I "maintain" it by:

1. Doing a "Copy all" of the error message from the Tests Explorer window.
2. Copying it into a blank file.
3. Splitting the "expected" and "actual" strings into separate files.
4. Using Beyond Compare to convince myself that the new "actual" is correct.
5. Updating the test by hand.

Not fun.

---
In reply to: [244508100](https://github.com/Microsoft/jschema/pull/62#discussion_r244508100) [](ancestors = 244508100)

Reply:
Dear god in heaven. There are several (unfortunately diverse) techniques demonstrated in the SARIF-SDK that would alleviate this. The best is to send all output to dedicated 'expected' and 'actual' directories. Then you can diff the dir for a large suite of tests. You can copy 'actual' files into 'expected' once you confirm that you like them. 

---
In reply to: [244508254](https://github.com/Microsoft/jschema/pull/62#discussion_r244508254) [](ancestors = 244508254,244508100)